txnbuild/timebounds.go

```diff
@@ -9,26 +9,28 @@ import (
 // what you want.
 const TimeoutInfinite = int64(0)
 
-// Timebounds represents the time window during which a Stellar transaction is considered valid.
+// TimeBounds represents the time window during which a Stellar transaction is considered valid.
 //
 // MinTime and MaxTime represent Stellar timebounds - a window of time over which the Transaction will be
 // considered valid. In general, almost all Transactions benefit from setting an upper timebound, because once submitted,
 // the status of a pending Transaction may remain unresolved for a long time if the network is congested.
 // With an upper timebound, the submitter has a guaranteed time at which the Transaction is known to have either
 // succeeded or failed, and can then take appropriate action (e.g. to resubmit or mark as resolved).
 //
-// Create a Timebounds struct using one of NewTimebounds(), NewTimeout(), or NewInfiniteTimeout().
-type Timebounds struct {
+// Create a TimeBounds struct using one of NewTimebounds(), NewTimeout(), or NewInfiniteTimeout().
+type TimeBounds struct {
 	MinTime  int64
 	MaxTime  int64
 	wasBuilt bool
 }
 
-// Validate for Timebounds sanity-checks the configured Timebound limits, and confirms the object was built
+// type Timebounds = TimeBounds
```

Comment:
Why is this commented? 
```suggestion
type Timebounds = TimeBounds
```

Reply:
I... I don't know :laughing: I did a ctrl+f and then un-commented it, but that disappeared somehow